### DISCUSSION TITLE
de/2-04-kata-3/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#54] [General]
# de/2-04-kata-3/

https://ai.lernos.org/de/2-04-kata-3/

**Comment by ghost** [comment]
Es wäre doch praktisch, wenn Referenzen auf die Grundlagenkapitel – wie z. B. im zweiten Absatz: "Lese Dir die ersten Abschnitte des Kapitels KI & Maschinelles Lernen durch." – nicht nur (wie aktuell) kursiv hervorgehoben sondern gleich als Link dargestellt würden.

**Reply by simondueckert** [reply]
Da sprichst du ein Dauerbrennerthema der lernOS Leitfäden an. In der Webversion ginge das prinzipiell. Das führt aber in unserem aktuellen technischen Setup dazu, dass z.B. in der Druckversion in der Fußnote ein Link zur Quelldatei des Kapitells (und nicht zum Titel des Kapitel mit Seitenzahl erscheint). Habe ich mit @maki-amw schon länger diskutiert und wir schauen uns das auf jeden Fall nochmal an.

**Reply by maki-amw** [reply]
Ja, da ist definitiv noch Verbesserungsbedarf und wir überlegen uns da nach der ersten Verprobungsphase für die Zukunft etwas. Zumindest über die OneNote-Vorlage konnten wir es durch eingebaute Direktverlinkungen zur Web-Version schon lösen, da sie nicht direkt mit der übergreifenden Produktionskette zusammenhängt. Also vielleicht ein guter Workaround für Dich / Euch, @schotul ?

**Reply by ghost** [reply]
Hallo @simondueckert, hallo @maki-amw,

mir ist das Thema beim Durcharbeiten der Webseiten aufgefallen. Insofern wäre mir bereits geholfen, wenn die Webversion entsprechend funktionieren würde.

Deinen Workaround, @maki-amw, habe ich nicht ganz verstanden: Meinst du damit, dass in einer Druckversion bei Anklicken eines Links zu einem anderen Kapitel die entsprechende Seite im Browser geöffnet wird? Falls ja, fände ich das nicht wirklich hilfreich.

Ich fände es wünschenswert, wenn in einer Druckversion bei Anklicken einer Verknüpfung zur entsprechenden Stelle im selben Dokument gesprungen würde. Hierdurch könnte man die Druckversion auch offline verwenden. Ich kann mir allerdings gut vorstellen, dass eine entsprechende technische Lösung nicht einfach zu finden ist.

**Comment by DoSchu-cowoxu** [comment]
In unserer Lerngruppe fiel in der darauffolgenden Kata 4 "Reflexion" auf, dass das persönliche Lernprojekt für lernOS-Neulinge zu kurz eingeführt und kaum konkret angeregt wird. Hier könnte beispielsweise aus dieser Kata im lernOS Sketchnoting Leitfaden https://cogneon.github.io/lernos-sketchnoting/de/0500_Kata_04/ etwas für zur Beschreibung für das persönliche Lernprojekt aufgegriffen werden.

**Comment by mkmikado** [comment]
Lese Dir  --> Lies Dir

**Comment by DoSchu-cowoxu** [comment]
In der Entwicklung eines Angebots auf der Basis des lernOS KI Leitfaden fiel uns auf: Es heisst ja "die KI", aber hier sprechen wir von "KI als Dialogpartner" - dabei wäre es ja deutlich sinniger von "KI als Dialogpartnerin" zu sprechen. 
( Wir haben in unserem Material ...DialogpartnerIn geschrieben. )
